buildSrc/src/main/groovy/stairway.java-conventions.gradle

```diff
@@ -14,7 +13,7 @@ repositories {
 
 dependencies {
     // For logging
-    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.36'
+    implementation group: 'org.slf4j', name: 'slf4j-api', version: '2.0.13'
```

Comment:
I was able to remove the stairctl subproject dependency exclusion by updating the slf4j version to the current release. This required changing the logback config files to xml, as groovy was deprecated a while ago due to security issues.